roundtables/week12/day-2-forms-worksheet-exquisite.md

```diff
@@ -0,0 +1,37 @@
+# Forms Worksheet and Exquisite React
+
+Today we are going to be discussing the Forms Worksheet and the Exquisite React project in a round table fashion. For this discussion, think of your instructor as a mediator and guide, rather than as a lecturer.
+
+For this activity, you are encouraged to do the following:
+
+* Whenever possible, you should use voice.
+* You should pose questions to the group, not the instructor.
+* Similarly, you are encouraged to answer questions, with the instructor available to correct misconceptions and provide answers when no one else has an answer.
+* Say something, even if you think you might be wrong! (Nobody throws a frisbee well on their first try)
+* When discussing code, you are encouraged to use VS Code Liveshare, and or screen sharing through zoom.
+* Take notes!
+
+## Materials
+[Forms Worksheet](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/React/exercises/forms-worksheet.md)
+
+[Exquisite React](https://github.com/Ada-C13/exquisite-react)
+
+
+## Prompts
+You are encouraged to copy these questions into a document and take notes during the discussion.
+
+### Forms Worksheet
+
+1. What is an event handler? Describe the event handler(s) you needed to make to manage the form data. What was the responsibility of these event handler(s)?
+1. How would you describe the relationship between the five input elements and their event handlers?
+    * Do you want to share your code to get feedback on specific parts?
```

Comment:
It might help students be more likely to share if this were worded to feel like a little bit stronger of a nudge/invitation. 
This would also be more consistent with how these have been worded in roundtables so far.
```suggestion
    * Share your code! Let's view 2 different implementations and discuss the elements and event handlers! Volunteers?
```